### PR TITLE
Delete address and tag fields

### PR DESCRIPTION
**Pull request by kangzongxian**
These are fields that are irrelevant to
our application.

Deleting these fields improves code quality,
and makes our codebase neater.

### CODE REVIEW

**Review by bryanhce — APPROVED** [review]
Nice work! Looks good to me.

File: src/main/java/seedu/guest/model/guest/Guest.java [hunk 1]
```diff
@@ -23,22 +18,17 @@ public class Guest {
...
-    private final Set<Tag> tags = new HashSet<>();
 
     /**
      * Every field must be present and not null.
      */
-    public Guest(Name name, Phone phone, Email email, DateRange dateRange, NumberOfGuests numberOfGuests,
-                 Address address, Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, numberOfGuests, address, tags);
+    public Guest(Name name, Phone phone, Email email, DateRange dateRange, NumberOfGuests numberOfGuests) {
+        requireAllNonNull(name, phone, email, numberOfGuests);
```
Comment (jehousoh) [inline src/main/java/seedu/guest/model/guest/Guest.java]: is dateRange supposed to be added here?

File: src/main/resources/view/GuestListCard.fxml [hunk 2]
```diff
@@ -27,9 +26,7 @@
         </Label>
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
       </HBox>
-      <FlowPane fx:id="tags" />
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
       <Label fx:id="numberOfGuests" styleClass="cell_small_label" text="\$numberOfGuests" />
       <Label fx:id="dateRange" styleClass="cell_small_label" text="\$dateRange" />
```
Comment (jehousoh) [inline src/main/resources/view/GuestListCard.fxml:31]: Should dateRange be above numberOfGuests for consistency?

**Review by jehousoh — CHANGES_REQUESTED** [review]
Looks good other than a few minor changes!

File: src/main/java/seedu/guest/model/guest/Guest.java (hunk 1 above)
Comment (kangzongxian) [reply src/main/java/seedu/guest/model/guest/Guest.java]: I do agree, fixed in latest commit!

File: src/main/resources/view/GuestListCard.fxml (hunk 2 above)
Comment (kangzongxian) [reply src/main/resources/view/GuestListCard.fxml:31]: Agree as well, fixed in latest commit!

**Review by jehousoh — APPROVED** [review]
lgtm!

**Review by JustinPeng13 — APPROVED** [review]
LGTM!

**Pull request merged**